### PR TITLE
Building binary for Linux using Nuitka

### PR DESCRIPTION
**Pull request by ZhymabekRoman**
Partially implements #54

### CODE REVIEW

**Comment by ZhymabekRoman** [comment]
I just noticed that we have `build.py` file :(

**Comment by ZhymabekRoman** [comment]
@Animenosekai, I am made some drafts to build binaries for Linux. So probably for now this PR is done for merge

File: .github/workflows/build_bin.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      - name: Cache Nuitka build
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/work/NUITKA_BUILD_CACHE
+          key: ${{ runner.os }}-nuitka-cache-${{ hashFiles('**/requirements.txt') }}-${{ github.run_number }}
+          restore-keys: ${{ runner.os }}-nuitka-cache-
+
+      - name: Building
+        run: |
+          NUITKA_CACHE_DIR="/home/runner/work/NUITKA_BUILD_CACHE" ./scripts/build_nuitka_linux.sh
```
Comment (Animenosekai) [inline .github/workflows/build_bin.yml]: Shouldn't it use the `build.py` script ? It's better to have everything in a single file

**Review by Animenosekai — COMMENTED** [review]
Should we use `build.py` ?

**Comment by ZhymabekRoman** [comment]
Well, maybe you are right. What is the point of having a build.py file?

**Comment by Animenosekai** [comment]
> Well, maybe you are right. What is the point of having a build.py file?

As you can see in the recent version of `build.py` it does way more than just building things, it helps manage the repo and helps maintaining it from a single file

**Comment by ZhymabekRoman** [comment]
Done

**Comment by Animenosekai** [comment]
Merged 👍

**Pull request merged**